notebooks/additional-methods.ipynb

```diff
@@ -0,0 +1,589 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    sys.path.insert(0, '/tonyear')
Line #9.    import sys
</pre><p>You can remove these lines</p><p> </p><p>Instead, run <code>pip install -e .</code>  in your Git repo and restart your Jupyter Kernel. It should pickup the local version of <code>tonyear</code>.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/carbonplan/ton-year/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>